.travis.yml

```diff
@@ -3,7 +3,8 @@ services: mongodb
 sudo: required
 node_js:
   - "8"
-  - "10"
+  - node
+cache: yarn
```

Comment:
Please remove this, I've had nothing but issues with the yarn cache and it really doesn't speed anything up. We could switch to npm's ci mode if we really wanted faster installs.